src/server/scripts/Northrend/IcecrownCitadel/boss_the_lich_king.cpp

```diff
@@ -1235,6 +1235,7 @@ class npc_tirion_fordring_tft : public CreatureScript
                     me->RemoveFlag(UNIT_NPC_FLAGS, UNIT_NPC_FLAG_GOSSIP);
                     me->SetWalk(true);
                     me->GetMotionMaster()->MovePoint(POINT_TIRION_INTRO, TirionIntro);
+                    me->SetWalk(false);
```

Comment:
uh? this should be called when chase movement starts, right?

Reply:
tirion never chases anything

Reply:
edited the comment